docs/src/emulation.md

```diff
@@ -3,35 +3,60 @@ id: emulation
 title: "Emulation"
 ---
 
-Playwright allows overriding various parameters of the device where the browser is running:
-- viewport size, device scale factor, touch support
-- locale, timezone
-- color scheme
-- geolocation
+Playwright allows overriding various parameters such as `viewportSize`, `deviceScaleFactor`, `locale`, `timezone`, `colorScheme`, `geolocation` and more.
 
-Most of these parameters are configured during the browser context construction, but some of them such as viewport size
-can be changed for individual pages.
+## Devices
+* langs: js, csharp, python
 
-<!-- TOC -->
+Playwright comes with a registry of device parameters for selected mobile devices. It can be used to simulate browser behavior on a specific mobile device. All pages created in the context will share the same device parameters.
```

Comment:
```suggestion
Playwright comes with a registry of device parameters for selected mobile devices. It can be used to simulate browser behavior on a specific mobile device. All tests will run with the specified device parameters.
```